### PR TITLE
#133: Update the exit command execution

### PR DESCRIPTION
**Pull request by Lolneilor**
Update the exit command execution to utilize exitCode instead of process.exit. This change aligns with the adaptation related to #130 . With this update, the  message will be displayed only after the command has fully completed its execution.

### CODE REVIEW

**Review by RicardoJRR-23 — DISMISSED** [review]

File: commands/user/create_account.js
Comment (vandro) [inline commands/user/create_account.js:1]: This file changes are not related to this scope.

File: commands/user/create_session.js [hunk 1]
```diff
@@ -46,9 +50,11 @@ async function login(user) {
     } else {
       console.error(chalk.redBright('Error:'), error.message || 'An error occurred');
     }
+    process.exitCode = 1;
   }
 }
 
 export default async function create_session() {
-  inquirer.prompt(questions).then(async (answers) => await login(answers));
```
Comment (vandro) [inline commands/user/create_session.js:53]: This change is not related to this scope.

File: commands/helpers/handle_error.js [hunk 2]
```diff
@@ -3,5 +3,5 @@ import chalk from 'chalk';
 // Helper function to handle errors
 export default function handleError(message) {
```
Comment (vandro) [inline commands/helpers/handle_error.js:4]: Please verify where this is being called and check if there is a `finally` on the try catch.

**Review by vandro — CHANGES_REQUESTED** [review]
Please see my comments.

**Comment by vandro** [comment]
Please change the version on `package.json`

File: commands/helpers/credentials.js [hunk 3]
```diff
@@ -87,7 +86,6 @@ export async function verify_auth_token() {
 
     if (!credentials?.LESS_TOKEN) {
       console.error(chalk.redBright('Error:'), '`LESS_TOKEN` not found in your environment variables. Please login using the `login` command and try again.');
-      process.exit(1);
```
Comment (RonyGomesK78) [inline commands/helpers/credentials.js:90]: you forgot to add the `process.exitCode=1`

**Review by RonyGomesK78 — DISMISSED** [review]

**Review by RonyGomesK78 — APPROVED** [review]

**Review by vandro — APPROVED** [review]

**Review by RonyGomesK78 — APPROVED** [review]

**Pull request merged**